build_env.sh

```diff
@@ -0,0 +1,17 @@
+# Node
```

Comment:
this file should be in your gitignore since the paths are all specific to the GSC